app/helpers/repository_helper.rb

```diff
@@ -23,8 +23,7 @@ def month_options
   end
 
   def year_options
```

Comment:
This helper started to look quite complicated for me. What do you think about replacing it by Rails' helpers? http://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html#method-i-select_year